### PR TITLE
Create report schedules doc

### PR DESCRIPTION
**Pull request by nancymic2**
Fixes b/273507785

Changes proposed in this pull request:

- https://pr-6877-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/attribution-reporting/report-schedules/
-
-

### CODE REVIEW

**Comment by nancymic2** [comment]
reviewed by Maud and Akash Nadan

File: site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+date: 2023-03-06
+updated: 2023-07-21
+authors:
+  - maudn
+---
+
+With the Attribution Reporting API, reports are not sent immediately after a user converts. Instead, the browser sends them with a delay.
+
+Note that regardless of report type, reports are only sent when the browser is running and online. After the first failure to send, the browser tries to send the report again after 5 minutes. After the second failure, the browser tries to send the report again after 15 minutes. After that, it's not sent.
+
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md:17]: Less of a writing question, more of a tech one—what's the solution to the problem? Regenerate a report when the sender is back online?

File: site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+### Report delays
+
+Aggregatable reports are sent by the browser to the ad tech provider or advertiser with a random delay between zero and 10 minutes, or with a small delay after the browser starts again. 
+
+## Event-level reports
+
+{% Aside %}
+What's described in this section is the default behavior. If you need to customize report scheduling behavior by shortening a report window, or if you need to filter out conversions happening after a certain window, you can use [custom report windows](/docs/privacy-sandbox/attribution-reporting/custom-report-windows/).
+{% endAside %}
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md:33]: Is it worth having this aside only one and stating that this page only describes the default behaviour?

File: site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+View-through conversion reports are sent by the browser to the ad tech approximately one hour after the view event the conversion is attributed to is no longer eligible for attribution.
```
Comment (anusmitaray) [inline site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md:35]: I'm confused by this sentence—should it have been two?

File: site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md (hunk 1 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md:17]: Great question. You can't generate a report, only the browser can, as far as I know. @maudnals what happens to the data in the report that is not sent?

File: site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md (hunk 2 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md:29]: Thanks. Done

File: site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md (hunk 3 above)
Comment (nancymic2) [reply site/en/docs/privacy-sandbox/attribution-reporting/report-schedules/index.md]: Clarified. thanks

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**